FirefoxPrivateNetworkVPNUITests/FirefoxPrivateNetworkVPNUITests.swift

```diff
@@ -79,4 +77,40 @@ class FirefoxPrivateNetworkVPNUITests: BaseTestCase {
         XCTAssertTrue(app.buttons["ReloadButton"].exists)
         XCTAssertTrue(app.webViews.textFields["Email"].exists)
     }
-}
+
+    func testSignInAsNonSubscribedUser() {
+        // The main screen is shown
+        waitForExistence(app.staticTexts["Mozilla VPN"], timeout: 5)
+        XCTAssertTrue(app.staticTexts["Mozilla VPN"].exists, "The main page is not loaded correctly")
+
+        // Tap on Get started
+        app.buttons["Get started"].tap()
+
+        // Wait for the FxASingIn page to be shown
+        waitForExistence(app.webViews.textFields["Email"], timeout: 10)
+        app.textFields["Email"].tap()
+        app.typeText("test-f5aefc1935@restmail.net")
```

Comment:
Could you create a variable to add this test account, something like:
let nonSubscribedEmailAccount = "test-f5aefc1935@restmail.net" 
Something similar to this [example](https://github.com/mozilla-mobile/firefox-ios/blob/main/XCUITests/SyncFAUITests.swift#L11)